blog/2021-05-06-documentation-information-architecture-sprint.md

```diff
@@ -0,0 +1,211 @@
+---
+tags:
+  - documentation
+  - information-architecture
+  - temporal
+  - site design
+posted_on_: 2021-05-06T00:00:00Z
+slug: docs-info-arch-2021
+title: 'Docs site April 2021 sprint'
```

Comment:
what are your thoughts on titling something more generally useful to others? imagine someone copy pasting in to their Slack, saying "we need to do this" - what does the title of that blogpost say?

```suggestion
title: 'Write Docs for Clear User Personas'
```